enhancements/update/change-management-and-maintenance-schedules.md

```diff
@@ -0,0 +1,1273 @@
+   Instead, updates are negotiated and planned far in advance. 
+1. The cluster workloads are not HA and unplanned drains are considered a business risk.
+1. To prevent surprises, the cluster lifecycle administrator sets the Assisted strategy on the worker MCP.
+1. In the `assisted` strategy change management policy, the lifecycle administrator configures `pausedUntil: "true"`
+   and the most recently rendered worker configuration in the policy's `renderedConfigsBefore: <current datetime>`.
+1. The MCO is being asked to ignite any new node or any rebooted node with the latest rendered configuration
+   before the present datetime. However, because of `pausedUntil: "true"`, it is also being asked not to 
+   automatically initiate that material change for existing nodes.
+1. The MCO metric for the MCP indicating the number of seconds remaining until changes can be initiated is `-1` - indicating
+   that there is presently no time in the future where it will initiate material changes. The operations team
+   has an alert configured if this value `!= -1`.
+1. The MCO metric for the MCP indicating that changes are pending is set because not all nodes are running
+   the most recent, rendered configuration. This is irrespective of the `renderedConfigsBefore` in the `assisted` 
+   configuration. Abstractly, it means, if change management were disabled, whether changes would be initiated.
+1. When the lifecycle administrator is ready to permit disruption, they set `pausedUntil: "false"`.
+1. The MCO sets the number of seconds until changes are permitted to `0`.
+1. The MCO begins to initiate worker node updates. This rollout abides by documented OpenShift constraints
+   such as the MachineConfigPool `maxUnavailable` setting.
+1. Though new rendered configurations may be created, the assisted strategy will not act until the assisted policy
+   is updated to permit a more recent creation date.
+   
+### API Extensions
+
+API Extensions are CRDs, admission and conversion webhooks, aggregated API servers,
+and finalizers, i.e. those mechanisms that change the OCP API surface and behaviour.
+
+- Name the API extensions this enhancement adds or modifies.
+- Does this enhancement modify the behaviour of existing resources, especially those owned
+  by other parties than the authoring team (including upstream resources), and, if yes, how?
+  Please add those other parties as reviewers to the enhancement.
+
+  Examples:
+  - Adds a finalizer to namespaces. Namespace cannot be deleted without our controller running.
+  - Restricts the label format for objects to X.
+  - Defaults field Y on object kind Z.
+
+Fill in the operational impact of these API Extensions in the "Operational Aspects
+of API Extensions" section.
+
+### Topology Considerations
+
+#### Hypershift / Hosted Control Planes
+
+In the HCP topology, the HostedCluster and NodePool resources are enhanced to support the change management strategies

```

Comment:
I would like to call out that:
1. HCP has both replace and inplace upgrade nodepools, and while that shouldn't matter for maint. schedule, they do upgrade differently, and
2. HCP differs from standalone in cert management, where the certs there will trigger creation of new nodes, and no rotation will happen if the upgrades are paused (due to a maint. schedule), so you run more risk of certs expiring down the line (although I don't know if they actually have certificate rotation handling for all certs today)